helm-chart/values.yaml

```diff
@@ -1,152 +1,133 @@
-  port: 7546
-  annotations: {}
 
 ingress:
   enabled: false
   className: ''
-  annotations:
-    {}
-    # kubernetes.io/ingress.class: nginx
-    # kubernetes.io/tls-acme: "true"
+  annotations: {}
   hosts:
     - host: chart-example.local
       paths:
         - path: /
           pathType: ImplementationSpecific
   tls: []
-  #  - secretName: chart-example-tls
-  #    hosts:
-  #      - chart-example.local
 
-resources:
-  {}
-  # It's recommended not to specify default resources and to leave this as a choice for the user.
-  # This also increases chances charts run on environments with little resources, such as Minikube.
-  #  If you do want to specify resources, uncomment use the following
-  # lines, as an example then and remove the curly braces after 'resources:' above.
-  # limits:
-  #   cpu: 100m
-  #   memory: 128Mi
-  # requests:
-  #   cpu: 100m
-  #   memory: 128Mi
+livenessProbe:
+  failureThreshold: 5
+  httpGet:
+    path: /health/liveness
+    port: jsonrpcrelay
+  initialDelaySeconds: 20
+  periodSeconds: 10
+  timeoutSeconds: 5
 
-autoscaling:
-  enabled: false
-  minReplicas: 1
-  maxReplicas: 100
-  targetCPUUtilizationPercentage: 80
-  # targetMemoryUtilizationPercentage: 80
+nameOverride: ''
 
 nodeSelector: {}
 
-tolerations: []
+podAnnotations: {}
 
-affinity: {}
+podSecurityContext: {}
 
 ports:
   name: jsonrpcrelay
   containerPort: 7546
 
-config:
-  # This value should be in decimal, not hexadecimal format.
-  CHAIN_ID: ''
-  MIRROR_NODE_URL: ''
-  LOG_LEVEL: ''
-  # Use config.local.HEDERA_NETWORK when running against a locally hosted hedera network
-  # local:
-  #   config.local.HEDERA_NETWORK should configured as a json set - {"$IPv4_ADDR_1:$PORT_1":"$ACCOUNT_ID_1","$IPv4_ADDR_2:$PORT_2":"$ACCOUNT_ID_2"}
-  #   HEDERA_NETWORK: {"127.0.01:50211":"0.0.3"}
-  #   HEDERA_NETWORK: ""
-  # Use config.hosted.HEDERA_NETWORK when running against a hosted Hedera network.  Valid options are `previewnet`, `testnet`, and `mainnet`
-  # hosted:
-  #   HEDERA_NETWORK: ""
-  OPERATOR_ID_MAIN: ''
-  OPERATOR_KEY_MAIN: ''
-  OPERATOR_ID_ETH_SENDRAWTRANSACTION: ''
-  OPERATOR_KEY_ETH_SENDRAWTRANSACTION: ''
-  DEFAULT_RATE_LIMIT: 200
-  TIER_1_RATE_LIMIT: 100
-  TIER_2_RATE_LIMIT: 200
-  TIER_3_RATE_LIMIT: 400
-  LIMIT_DURATION: 60000
-  HBAR_RATE_LIMIT_TINYBAR: 5000000000
-  HBAR_RATE_LIMIT_DURATION: 60000
-  ETH_GET_LOGS_BLOCK_RANGE_LIMIT: 1000
-  RATE_LIMIT_DISABLED: false
-  DEV_MODE: false
-  GAS_PRICE_TINY_BAR_BUFFER: 10000000000
-  MIRROR_NODE_RETRIES: 3
-  MIRROR_NODE_RETRY_DELAY: 250
-  MIRROR_NODE_LIMIT_PARAM: 100
-  CLIENT_TRANSPORT_SECURITY: false
-  INPUT_SIZE_LIMIT: 1
-  ETH_CALL_CACHE_TTL: 200
-  CONSENSUS_MAX_EXECUTION_TIME: 15000
-  SDK_REQUEST_TIMEOUT: 10000
-  CONTRACT_QUERY_TIMEOUT_RETRIES: 3
-  SUBSCRIPTIONS_ENABLED: false
-  ETH_CALL_DEFAULT_TO_CONSENSUS_NODE: true
+replicaCount: 2
+
+resources:
+  limits:
+    cpu: 1100m
+    memory: 350Mi
+  requests:
+    cpu: 700m
+    memory: 64Mi
```

Comment:
You should consider setting `requests==limits` so the pods gets guaranteed [Quality of Service](https://kubernetes.io/docs/tasks/configure-pod-container/quality-service-pod/). We've found cases where our nodejs gets throttled but also never even gets scheduled onto a node that would have sufficient available CPU.

Reply:
Updated